auto_round/autoround.py

```diff
@@ -2444,6 +2495,11 @@ def quantize_block(self, block, input_ids, input_others, q_input=None, device=to
         with torch.no_grad():
             unwrapper_block(block, best_params)
         if self.enable_quanted_input:
+            if is_nv_fp(self.act_data_type) and any(BackendDataType.NV_FP.value in format_ for format_ in self.formats):
```

Comment:
what's the meaning of BackendDataType,  is it possible to delete it and just use str